### DISCUSSION TITLE
Xbox Rich Presence ideas!

### DISCUSSION THREAD

**Discussion by MrCoolAndroid** [MrCoolAndroid/Xbox-Rich-Presence-Discord#6] [Ideas]
# Welcome to the Ideas discussion
Post in here any ideas or things you want me to add to the app!

**Comment by ckacadv** [comment]
automatic sign in when u open the app

**Reply by MrCoolAndroid** [reply]
Thanks! I can make it happen, I'm trying to get a bunch of other suggested stuff too into one single update

**Comment by HQv2** [comment]
Is it possible to show the game name here, instead of Xbox?

![image](https://github.com/MrCoolAndroid/Xbox-Rich-Presence-Discord/assets/42633588/d4d4401f-b2fa-4981-b402-6d56481762a8)

**Reply by MrCoolAndroid** [reply]
Hello there! Sadly at the moment it's not possible because that's how the Discord presence works, there's another way to show the game name instead but it includes using your Discord token which can lead to security issues

**Comment by ComixGaming248** [comment]
In the future a moblie app that runs in the backround even when the phone is off (if you give the app the perrmission to).
Of course you'll need discord and xbox app installed for it to function. I know this is pretty out there but I've been thinking about it for a while.

**Reply by MrCoolAndroid** [reply]
Hey there! There is an issue when making rich presence on Android, you need to use the user's Discord token, which can lead to privacy and security issues, there is an app named [Kizzy](https://github.com/dead8309/Kizzy) that does that, if you want to check it out

**Reply by ComixGaming248** [reply]
Looked into it. Thanks for the reccomendation. :)
